smart-contracts/test/Lock/purchaseFor.js

```diff
@@ -16,252 +16,255 @@ contract('Lock / purchaseFor', accounts => {
     locks = await deployLocks(unlock, accounts[0])
   })
 
-  describe('when the contract has a public key release', () => {
-    it('should fail if the price is not enough', async () => {
-      await shouldFail(
-        locks.FIRST.purchase(0, accounts[0], web3.utils.padLeft(0, 40), [], {
-          value: Units.convert('0.0001', 'eth', 'wei'),
-        }),
-        'NOT_ENOUGH_FUNDS'
+  it('should fail if the price is not enough', async () => {
```

Comment:
Hmm. Removibg this ine "describe" statement is mak8ng it look kije Ive dobe waaay more in this fike than I actually did. I might restore it for the purpose of making reviewing easier.

Reply:
done.